visualizer_drag_gradio.py

```diff
@@ -0,0 +1,866 @@
+import os
+import os.path as osp
+from argparse import ArgumentParser
+from functools import partial
+
+import gradio as gr
+import numpy as np
+import torch
+from PIL import Image
+
+import dnnlib
+from gradio_utils import (ImageMask, draw_mask_on_image, draw_points_on_image,
+                          get_latest_points_pair, get_valid_mask,
+                          on_change_single_global_state)
+from viz.renderer import Renderer, add_watermark_np
+
```

Comment:
hi @XingangPan if you'd like I've upload the models to a repository on Hugging Face

```python 
# download models from HuggingFace hub
from huggingface_hub import snapshot_download
model_dir = Path('./checkpoints')
snapshot_download('radames/DragGan', repo_type='model', local_dir=model_dir)
```

Reply:
I'll be happy to move the repo to @XingangPan